File: Logic & Design/Visual Studio Code/Loops/Python/bug_collector2.py
'''
Programmer:         Corey Jenkins
Date:               February 4, 2023
Filename:           bug_collector2.py
Purpose:            To create a program using a module, that runs a loop of seven days and putting in how many bugs you've collected a day.
'''

def main():
    # Function total_bugs
    total_bugs()
    
def total_bugs():
    # Set the integer total to 0
    total = 0
    
    # Integer days. To collect bugs up to seven days in a for loop.
    # The difference between this program and the program bug_collector.py is that the total is calculated every day in the loop.
    for days in range(1, 8): # To loop day 1 through seven
        
        bugs = int(input("Bug(s) collected:"))
        total = (total + bugs)
        print("Bug(s) collected on day",days,":",bugs)
        print("Total bug(s) collected:",total)
        
    print("The total amount collected in seven days:",total,".")
    
main()

File: Logic & Design/Visual Studio Code/Loops/Java/BugCollector2.java
/*
 * Programmer:      Corey Jenkins
 * Date:            February 4, 2022
 * Filename:        BugCollector2.java
 * Purpose:         To create a program using a module, that runs a loop of seven days and putting in how many bugs you've collected a day.
 */

// Import the scanner for java.
import java.util.Scanner;

public class BugCollector2 {
    public static void main(String[] args) {
        showTotal();
    }

    public static void showTotal() {
        // Place the scanner for input.
        Scanner keyboard = new Scanner(System.in);

        // Integers bugs and days. Days will be used to loop.
        int bugs, days;

        // Another integer for total amount each day. It is set to 0
        int total = 0;

        for (days = 1; days < 7; days++) {
            System.out.println("Bug(s) collected:");
            bugs = keyboard.nextInt();

            // Formula.
            total = (total + bugs);

            System.out.println("Bug(s) collected for day " + days + ": " + bugs);
            System.out.println("Total amount: " + total);
        }
        // Sentence has been re-worked to be made sensical.
        System.out.println("The total amount collected in seven days: " + total + ".");

    }
}
